src/frontend/src/components/public/PIAFormTabs/accuracyRetention/index.tsx

```diff
@@ -15,7 +15,13 @@ import ViewComments from '../../../common/ViewComment';
 import { PiaSections } from '../../../../types/enums/pia-sections.enum';
 import Radio from '../../../common/Radio';
 
-export const AccuracyCorrectionAndRetention = () => {
+interface AccuracyCorrectionAndRetentionProps {
```

Comment:
I think we should have interfaces live in their own separate files and be called in instead?

Reply:
Switching to a separate interface for the calls has definitely made the code cleaner. Thank you for suggesting this approach!